programs/voter-stake-registry/src/cpi_instructions.rs

```diff
@@ -348,7 +358,9 @@ pub fn withdraw_mining<'a>(
     amount: u64,
     owner: &Pubkey,
     signers_seeds: &[&[u8]],
+    delegate_wallet_addr: &Pubkey,
 ) -> ProgramResult {
+    // TODO: Maybe it necessary to put `delegate_wallet_addr` into metadata
```

Comment:
You will not be reading the `delegate` account on the rewards side. So just passing it as a pubkey is fine.